tests/pom/common/Comments.ts

```diff
@@ -1,6 +1,51 @@
 import type { Locator, Page } from '@playwright/test';
+import { ConfirmationDialog } from './ConfirmationDialog';
 
-export class ActivityComments {
+export class Comment {
+  static selector = '.comment';
+  readonly page: Page;
+  readonly el: Locator;
+  readonly resolveBtn: Locator;
+  readonly editBtn: Locator;
+  readonly removeBtn: Locator;
+  readonly confirmationDialog: ConfirmationDialog;
+
+  constructor(page: Page, el: Locator) {
+    this.page = page;
+    this.el = el;
+    this.resolveBtn = el.getByRole('button', { name: 'Resolve comment' });
+    this.editBtn = el.getByRole('button', { name: 'Edit comment' });
+    this.removeBtn = el.getByRole('button', { name: 'Remove comment' });
+    this.confirmationDialog = new ConfirmationDialog(page, 'Remove comment');
+  }
+
+  focus() {
+    return this.el.click();
+  }
+
+  resolve() {
+    return this.resolveBtn.click();
```

Comment:
Would it make sense to 'bundle' in some of these both the assertion that the action has been completed.